public/locales/pt/editor.json

```diff
@@ -0,0 +1,16 @@
+{
+	"editor-desktop-optimized": "Esse site é otimizado para o desktop",
+	"editor-visit-desktop": "Por favor visite readme.so em um desktop para criar o seu readme!",
+	"nav-download": "Download",
+	"download-readme-generated": "Readme Gerado!",
+	"download-reach-out": "Obrigado por usar o readme.so! Sinta-se livre para entrar em contato comigo no",
+	"download-feedback": "com qualquer feedback.",
+	"download-coffee": "Se você achou esse projeto útil, considere me dar um suporte com uma xícara de café!",
+	"editor-column-editor": "Editor",
+	"editor-select": "Selecione uma seção da barra lateral a esquerda para editor os conteúdos",
+	"preview-column-preview": "Pré visualização",
+	"preview-column-raw": "Código fonte",
+	"section-column-section": "Seções",
+	"section-column-click-edit": "Clique em uma seção abaixo para editar os conteúdos",
+	"section-column-click-add": "Clique em uma seção abaixo para adicioná-la no seu readme"
```

Comment:
```suggestion
	"section-column-click-add": "Clique em uma seção abaixo para adicioná-la ao seu readme"
```

Reply:
This one don't make much difference IMO

Reply:
If it doesn't make it worse for pt-BR, please accept it then — for pt-PT, "adicionar no" is plain wrong.

Reply:
Nice, going to accept it :)